src/test/skript/tests/syntaxes/effects/EffHealth.sk

```diff
@@ -20,4 +20,7 @@ test "health effect":
 	assert health of {_m} is 1.5 with "heal cow failed"
 	heal {_m} by 99
 	assert health of {_m} is 5 with "heal cow failed"
+	wait 1 tick
+	damage {_m} by 2 hearts with damage cause fall
+	assert health of {_m} is 3 with "damage cow failed"
```

Comment:
We must wait, because:
![image](https://github.com/user-attachments/assets/93a40b99-359f-472f-ba53-b3df5001745c)
![image](https://github.com/user-attachments/assets/3c0aa07c-fe9f-43c0-8978-ca18cb5308a0)
It just marked the entity to be damaged, the damage will be delayed

Reply:
And there is something wrong with `parse if` section:
![image](https://github.com/user-attachments/assets/1f52fbbb-3e88-418a-b1ae-99f31539324d)
![image](https://github.com/user-attachments/assets/90c53f73-b0f4-4513-88dd-a37d284752a7)

I can only upload the `parse if` version and wait for it to be fixed

Reply:
You cannot wait, because anything after the wait will run after the test runner is finished, essentially making it useless.

Reply:
> You cannot wait, because anything after the wait will run after the test runner is finished, essentially making it useless.

Yes, so I removed it. But idk how to fix the `mark` problem, bcs the damage is delayed